FrameworkSystem/private/monitoring/ServiceInterface.py

```diff
@@ -30,15 +38,12 @@ def __createCatalog( self ):
     """
     Creates a Monitoring catalog connector
     """
-    from DIRAC.FrameworkSystem.private.monitoring.MonitoringCatalog import MonitoringCatalog
```

Comment:
I read that "This module exposes singleton"

I don't know what can go wrong here. I am just saying that when I was (and still am) fighting with these things, putting the import inside the function/method was the only way for some things to work, because if it was in the global file, they would be imported during the initialization while the info are only available in the end.
I am not saying that it is the case in this particular place, I am just pointing out that if you move it because you follow blindly what PyLint says, without knowing what you are doing here, we might have surprises.